cpp/dolfinx/fem/assemble_vector_impl.h

```diff
@@ -196,10 +200,10 @@ void _lift_bc_cells(
         if (bc_markers1[jj])
         {
           const T bc = bc_values1[jj];
-          if (!x0.empty())
-            be -= Ae.col(bs1 * j + k) * scale * (bc - x0[jj]);
-          else
-            be -= Ae.col(bs1 * j + k) * scale * bc;
+          const T _x0 = x0.empty() ? 0.0 : x0[jj];
+          // be -= Ae.col(bs1 * j + k) * scale * (bc - _x0);
```

Comment:
Commented code also here, probably in most assembly functions?

Reply:
Yes, for readability to the reader doesn't have to reverse engineer the operaion.

Reply:
I've left this one because it's helpful to summarise the 'math' operation.